src/libs/actions/Report.js

```diff
@@ -266,22 +266,28 @@ function fetchChatReportsByIDs(chatList) {
         .then(({reports}) => {
             Log.info('[Report] successfully fetched report data', true);
             fetchedReports = reports;
-
+            return Promise.all(_.map(fetchedReports, updateIOUReportData));
+        })
+        .then((data) => {
             // Process the reports and store them in Onyx. At the same time we'll save the simplified reports in this
             // variable called simplifiedReports which hold the participants (minus the current user) for each report.
             // Using this simplifiedReport we can call PersonalDetails.getFromReportParticipants to get the
             // personal details of all the participants and even link up their avatars to report icons.
-            const simplifiedReports = {};
-            _.each(fetchedReports, (report) => {
-                const key = `${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`;
+            const reportIOUData = {};
+            _.each(fetchedReports, (report, index) => {
+                const iouReportData = data[index] || {state: 1, cachedTotal: 0};
```

Comment:
I think IOUReport data is always falling back to the default here: 

![Screenshot 2021-03-25 at 18 47 20](https://user-images.githubusercontent.com/10736861/112527148-a228e080-8d9a-11eb-82df-231c78984834.png)

Reply:
I cutoff the URL from .com, but that's report 796